static/src/javascripts/projects/common/modules/experiments/tests/facebook-like-prompt.js

```diff
@@ -0,0 +1,176 @@
+                    status: true, // check login status
+                    cookie: true, // enable cookies to allow the server to access the session
+                    xfbml: true // parse XFBML
+                });
+            });
+        };
+
+        var progressBarTest = function (editions, options) {
+            return function () {
+                var edition = editions[config.page.edition];
+
+                getLikes(username)
+                    .then(function (likes) {
+                        var remainingLikes = edition.target - likes;
+                        var percentageComplete = (likes / edition.target) * 100;
+
+                        new Message('facebook-like-prompt', {
+                            pinOnHide: false,
+                            siteMessageCloseBtn: 'hide'
+                        }).show(getFacebookLikePromptProgressBarTemplate(edition, likes, remainingLikes, percentageComplete, options));
+
+                        loadFacebookSdk();
+                    });
+            };
+        };
+
+        this.variants = [
+            {
+                id: 'control',
+                test: function () {}
+            },
+            {
+                id: 'simple',
+                test: function () {
+                    new Message('facebook-like-prompt', {
+                        pinOnHide: false,
+                        siteMessageCloseBtn: 'hide'
+                    }).show(template(
+                        '<p class="site-message__message" id="site-message__message">{{messageText}}</p>' +
+                        '<div class="site-message--facebook-like-prompt__like">' +
+                            '<div class="fb-like"' +
+                                 'data-href="https://www.facebook.com/' + username + '"' +
+                                 'data-layout="button_count"' +
+                                 'data-action="like"' +
+                                 'data-show-faces="true"' +
+                                 'data-share="false"></div>' +
+                        '</div>',
+                        {
+                            messageText: 'Like the Guardian on Facebook'
+                        }
+                    ));
+
+                    loadFacebookSdk();
+                }
+            },
+            {
+                id: 'progress-bar--high-target',
+                test: progressBarTest({
+                    'UK': { target: 10000000, textTarget: '10 million' },
+                    'AU': { target: 500000 },
+                    'US': { target: 500000 }
+                }, { showIncompleteLabelInsideProgressBar: true })
+            },
+            {
+                id: 'progress-bar--low-target',
+                test: progressBarTest({
+                    'UK': { target: 5000000, textTarget: '5 million' },
+                    'AU': { target: 400000 },
+                    'US': { target: 300000 }
+                })
```

Comment:
Is it a fair test if the US target is higher than AU for the high target
variant, but vice versa for the low target variant?

On Wed, 22 Apr 2015 at 10:54 Chris Mulholland <notifications@github.com>
wrote:

> In
> static/src/javascripts/projects/common/modules/experiments/tests/facebook-like-prompt.js
> <https://github.com/guardian/frontend/pull/8958#discussion_r28858402>:
>
> > +            },
> > +            {
> > +                id: 'progress-bar--high-target',
> > +                test: progressBarTest({
> > +                    'UK': { target: 10000000, textTarget: '10 million' },
> > +                    'AU': { target: 500000 },
> > +                    'US': { target: 500000 }
> > +                }, { showIncompleteLabelInsideProgressBar: true })
> > +            },
> > +            {
> > +                id: 'progress-bar--low-target',
> > +                test: progressBarTest({
> > +                    'UK': { target: 5000000, textTarget: '5 million' },
> > +                    'AU': { target: 400000 },
> > +                    'US': { target: 300000 }
> > +                })
>
> Lets do this:
>
> US is a bit more extreme, but there's certainly more growth potential
> there.
>
> {
>
>    - id: 'progress-bar--high-target',
>    - test: progressBarTest({
>    - 'UK': { target: 10000000, textTarget: '10 million' },
>    - 'AU': { target: 750000 },
>    - 'US': { target: 1000000 }
>    - }, { showIncompleteLabelInsideProgressBar: true })
>    - },
>    - {
>    - id: 'progress-bar--low-target',
>    - test: progressBarTest({
>    - 'UK': { target: 5000000, textTarget: '5 million' },
>    - 'AU': { target: 300000 },
>    - 'US': { target: 250000 }
>    - })
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/8958/files#r28858402>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396